gce_rescue/utils.py

```diff
@@ -206,6 +206,8 @@ def read_input(msg: str) -> None:
   """Read user input if --force is not provided."""
   print(msg, end='')
   input_answer = input()
+  input_answer = input_answer.strip()
   if input_answer.upper() != 'Y':
     print('Cancelled.')
```

Comment:
This line seems to be irrelevant right now, dont you think ? @tomerlf1

Reply:
Yes, this was left there by mistake. thanks for noticing